cassiopeia/solver/VanillaGreedySolver.py

```diff
@@ -53,9 +53,10 @@ def __init__(
         self,
         missing_data_classifier: Callable = missing_data_methods.assign_missing_average,
         prior_transformation: str = "negative_log",
+        collapse_tree: bool = True
```

Comment:
Please add documentation for this `collapse_tree` argument, as with the other `GreedySolver` (sub)classes